mobile/src/main/java/ch/epfl/sdp/mobile/ui/speech_recognition/SpeechRecognizer.kt

```diff
@@ -0,0 +1,3 @@
+package ch.epfl.sdp.mobile.ui.speech_recognition
+
+class SpeechRecognizer {}
```

Comment:
Why do you have an empty class here ?

Reply:
I don't think I'll need it. It was just a place holder. I'll remove it.